src/dodona/junit/JSONListener.java

```diff
@@ -1,28 +1,24 @@
 package dodona.junit;
 
-import dodona.feedback.Status;
-import dodona.feedback.StartTab;
-import dodona.feedback.StartContext;
-import dodona.feedback.StartTestcase;
-import dodona.feedback.CloseTab;
-import dodona.feedback.CloseContext;
-import dodona.feedback.CloseTestcase;
-import dodona.feedback.Message;
-import dodona.feedback.AppendMessage;
-import dodona.feedback.EscalateStatus;
+import dodona.feedback.*;
```

Comment:
We don't use everything, so I prefer not to use a `*`.

Reply:
This was not intentional; my IDE performs import cleanup whenever I save a file. I have reversed this.